target/linux/ramips/dts/Newifi-D2.dts

```diff
@@ -0,0 +1,165 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "pandorabox,newifi-d2", "mediatek,mt7621-soc";
+	model = "Newifi-D2";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x10000000>;
```

Comment:
should be <0x0 0x1c000000>, <0x20000000 0x4000000>
otherwise it won't recognize all 512M ram